tests/test-sources/plugins/lsp/conform.nix

```diff
@@ -0,0 +1,16 @@
+{
+  empty = {
+    plugins.conform-nvim.enable = true;
+  };
+
```

Comment:
Could you also add a test with all the default options? (I think this is this: https://github.com/stevearc/conform.nvim/blob/253878436e2b6d73dfd91ccf0ac12d04cc683d34/doc/conform.txt#L14-L91)